src/components/events/Event/index.js

```diff
@@ -23,16 +22,25 @@ module.exports = React.createClass({
     };
   },
 
-  // componentWillMount: function() {
-  //   var _this = this;
-  //   var backgroundImage = document.createElement('img');
-  //   backgroundImage.src = this.props.backgroundImageUrl;
-  //   backgroundImage.onload = function() {
-  //     _this.setState({
-  //       blurredImage: effect.blurImage(backgroundImage, 30)
-  //     });
-  //   };
-  // },
+  // () -> ()
+  getInitialState: function() {
+    return {
+      blurredBackgroundImage: ''
+    };
+  },
+
+  // () -> ()
+  componentWillMount: function() {
+    var _this = this;
+    var backgroundImage = document.createElement('img');
+    backgroundImage.setAttribute('crossorigin', 'anonymous');
+    backgroundImage.src = this.props.backgroundImageUrl;
+    backgroundImage.onload = function() {
+      _this.setState({
+        blurredBackgroundImage: effect.blurImage(backgroundImage, 30)
+      });
+    };
+  },
 
   // () -> XJS
```

Comment:
What are they for? We all know what the render function does and accepts. Why would we comment it? I think we only need to add comment if there is something strange about code. Otherwise generally code should be shelf explanatory. Am I missing something? https://github.com/everydayhero/styleguide/blob/master/Javascript.md